### PR TITLE
Quick step to contribute to the project

### PR DESCRIPTION
**Pull request by LChettia**
This is in relation to #1

### CODE REVIEW

**Comment by LChettia** [comment]
Hi please review this

**Comment by LChettia** [comment]
@klakshmin can you check and review this

**Comment by LChettia** [comment]
@htaing this closed however check if this is reflecting in your notification

**Comment by htaing** [comment]
looks good

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# How to Contribute
```
Comment (htaing) [inline CONTRIBUTION.md:1]: nice use of spelling

**Comment by LChettia** [comment]
@klakshmin please review this again

**Pull request merged**